lib/cli/triage.js

```diff
@@ -47,12 +41,32 @@ module.exports = async () => {
       )
     ).find(Boolean) || null;
 
+  const isChinaUser = (() => {
```

Comment:
Is there a reason why we cannot use a similar util from `@serverless/utils` package?

Reply:
This a triage, and I would like to minimize dependencies for few reasons:
- Sharing logic with Serverles Framework, may introduce unwanted side effect. e.g. when we change anything in `@serverless/utils` we rather assume we change things only for Framework and dashboard plugin, while accidently we may change behavior of a triage which is crucial for Tencent (that has totally separate development flow)
- It's better for triage transparency to have that logic inline, it can also be tweaked specifically for Tencent needs.
- performance, I'd require other modules, only if we have a very good reason for it

Reply:
This a triage, and I would like to minimize dependencies for few reasons:

- Sharing logic with Serverles Framework, may introduce unwanted side effect. e.g. when we change anything in @serverless/utils we rather assume we change things only for Framework and dashboard plugin, while accidently we may change behavior of a triage which is crucial for Tencent (that has totally separate development flow)
- It's better for triage transparency to have that logic inline, it can also be tweaked specifically for Tencent needs.
- performance, I'd require other modules, only if we have a very good reason for it

Reply:
Thanks for explanation, that's what I was thinking but wanted to be sure 👍